spec/factories/bills.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :bill, class: LedgerSync::Bill do
+    references_one :account
+
+    sequence(:memo) { |n| "Memo #{n}" }
```

Comment:
Have you ever used the `Faker` gem? It could be nice for parts like the memo. But this is also fine for now

Reply:
I have used it.  We could start to do it.  I left it like this though in case you needed to line up errors with objects in the ledger itself or something.  Though uniqueness becomes an issue since tests will re-use the same names.  Happy either way.